src/main/cpp/util/path_windows.cc

```diff
@@ -230,9 +239,10 @@ bool AsWindowsPath(const std::string& path, std::string* result,
 
   if (path[0] == '\\') {
     // This is an absolute Windows path on the current drive, e.g. "\foo\bar".
-    mutable_path = std::string(1, GetCurrentDrive()) + ":" + path;
+    std::basic_string<char_type> drive(1, GetCurrentDrive());
+    drive.push_back(':');
```

Comment:
I'm curious, is using push_back more efficient than the string concat in the old code?

Reply:
The problem is, I cannot use `":"` and `L":"` at the same time. But `push_back(':')` works for both wstring and string. This is an workaround to make the code work for both situation. ;)